File: src/main/java/com/youngjun/auth/storage/db/core/token/TokenJpaRepository.java
package com.youngjun.auth.storage.db.core.token;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface TokenJpaRepository extends JpaRepository<TokenEntity, Long> {

    List<TokenEntity> findByAuthId(Long authId);

    List<TokenEntity> findByRefreshToken(String refreshToken);

    void deleteByAuthId(Long authId);
}

File: src/main/kotlin/com/youngjun/auth/storage/db/core/token/TokenJpaRepository.kt
package com.youngjun.auth.storage.db.core.token

import org.springframework.data.jpa.repository.JpaRepository

interface TokenJpaRepository : JpaRepository<TokenEntity, Long> {
    fun findByAuthId(authId: Long): List<TokenEntity>

    fun findByRefreshToken(refreshToken: String): List<TokenEntity>

    fun deleteByAuthId(authId: Long)
}
